packages/vuetify/src/components/VSlider/slider.ts

```diff
@@ -158,10 +158,10 @@ export const useSlider = ({
   const max = computed(() => parseFloat(props.max))
   const step = computed(() => props.step > 0 ? parseFloat(props.step) : 0)
```

Comment:
I'd change this to:

```ts
const step = computed(() => parseFloat(props.step > 0 ? props.step : 0))
```

So that it always returns a string.

Reply:
It's probably better to keep it as `number` type, because there are other places using it to do arithmetic calculation.

Instead, I put `toString` as part of `getDecimals` function in [new commit](43e1a317a35f2a5807469beae2dc5ad99bd5a0b4)

![image](https://user-images.githubusercontent.com/5698884/220242840-1f2e23fa-e508-47e1-8191-58a86726b241.png)